docs/reference/generated/accordion-root.json

```diff
@@ -20,27 +21,31 @@
       "description": "Allows the browser’s built-in page search to find and expand the panel contents.\n\nOverrides the `keepMounted` prop and uses `hidden=\"until-found\"`\nto hide the element without removing it from the DOM."
     },
     "openMultiple": {
-      "type": "boolean",
+      "type": "boolean | undefined",
       "default": "true",
+      "required": true,
       "description": "Whether multiple items can be open at the same time."
     },
     "disabled": {
-      "type": "boolean",
+      "type": "boolean | undefined",
       "default": "false",
+      "required": true,
```

Comment:
This is a bit strange. It says it's required, but we allow undefined as value. Is there a way to change the props that have default values not to be flagged as required?